casepro/backend/junebug.py

```diff
@@ -40,6 +46,74 @@ def get_addresses(self, uuid):
         return (
             a['address'] for a in addresses if a.get('address') is not None)
 
+    def get_identities(self, **kwargs):
+        '''Get the list of identities filtered by the given kwargs.'''
+        url = '%s/api/v1/identities/search/?' % (self.base_url)
+        params = {}
+        for key, value in kwargs.iteritems():
+            params[key] = value
```

Comment:
You can delete the three lines above and just put `**params` in the function signature.